### PR TITLE
V.0

### PR DESCRIPTION
**Pull request by kem247**
Resolves issue #5

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react"
 import { Link } from "gatsby"
 
 import Layout from "../components/layout"
-import Image from "../components/image"
+// import Image from "../components/image"
```
Comment (mewtaylor) [inline src/pages/index.js:5]: I think if we're not going to use the component, it should just be deleted rather than commented out.

File: src/pages/about.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react"
```
Comment (mewtaylor) [inline src/pages/about.js:1]: Do we need this page? I think it's not necessary, and think we could remove it, unless I'm misremembering something.

File: src/components/layout.js [hunk 3]
```diff
@@ -25,6 +25,16 @@ const Layout = ({ children }) => {
 
   return (
     <>
+      <Helmet
```
Comment (mewtaylor) [inline src/components/layout.js:28]: I think that metadata/helmet stuff should go in the Layout, above the header field. Would it be possible to take this part out of the PR and just focus on the page rendering correctly, without SEO stuff? I can work on the header/helmet stuff as part of migrating the existing homepage.

File: src/components/header/index.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types"
 import React from "react"
 
 const Header = ({ siteTitle }) => (
-  <header
+  <div
```
Comment (mewtaylor) [inline src/components/header/index.js:6]: See my comment about the Helmet – is it possible to revert these changes, and focus testing on the components inside page 2 only?

File: src/components/_tests_/header.test.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
```
Comment (mewtaylor) [inline src/components/_tests_/header.test.js:1]: I think that this v0 can just test the rendering of the component added to page 2, and does not need to include tests for existing components that come with Gatsby

File: setup-test-env.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import { createSerializer } from "jest-emotion"
+import * as emotion from "@emotion/core"
```
Comment (mewtaylor) [inline setup-test-env.js:2]: I thought when we chatted on Thursday we had decided not to use emotion, correct? would it be possible to remove it?

Comment (kem247) [reply setup-test-env.js:2]: Hi I realized that was how I was able to make snapshots, so I made the decision to use it.

**Comment by mewtaylor** [comment]
Closing as no longer relevant, per @kem247 suggestion!
